src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse2.cs

```diff
@@ -874,536 +1096,692 @@ public static class Sse2
         }
 
         /// <summary>
-        /// __m128i _mm_sad_epu8 (__m128i a,  __m128i b)
+        /// __m128i _mm_sad_epu8 (__m128i a,  __m128i b);
+        ///   PSADBW xmm, xmm/m128
         /// </summary>
         public static Vector128<long> SumAbsoluteDifferences(Vector128<byte> left, Vector128<byte> right) => SumAbsoluteDifferences(left, right);
 
         /// <summary>
-        /// __m128i _mm_shuffle_epi32 (__m128i a,  int immediate)
+        /// __m128i _mm_shuffle_epi32 (__m128i a,  int immediate);
+        ///   PSHUFD xmm, xmm/m128, imm8
         /// </summary>
         public static Vector128<int> Shuffle(Vector128<int> value, byte control) => Shuffle(value, control);
         /// <summary>
-        /// __m128i _mm_shuffle_epi32 (__m128i a,  int immediate)
+        /// __m128i _mm_shuffle_epi32 (__m128i a,  int immediate);
+        ///   PSHUFD xmm, xmm/m128, imm8
         /// </summary>
         public static Vector128<uint> Shuffle(Vector128<uint> value, byte control) => Shuffle(value, control);
         /// <summary>
-        /// __m128d _mm_shuffle_pd (__m128d a,  __m128d b, int immediate)
+        /// __m128d _mm_shuffle_pd (__m128d a,  __m128d b, int immediate);
+        ///   SHUFPD xmm, xmm/m128, imm8
         /// </summary>
         public static Vector128<double> Shuffle(Vector128<double> left, Vector128<double> right, byte control) => Shuffle(left, right, control);
-        
+
         /// <summary>
-        /// __m128i _mm_shufflehi_epi16 (__m128i a,  int immediate)
+        /// __m128i _mm_shufflehi_epi16 (__m128i a,  int immediate);
+        ///   PSHUFHW xmm, xmm/m128, imm8
         /// </summary>
         public static Vector128<short> ShuffleHigh(Vector128<short> value, byte control) => ShuffleHigh(value, control);
         /// <summary>
-        /// __m128i _mm_shufflehi_epi16 (__m128i a,  int control)
+        /// __m128i _mm_shufflehi_epi16 (__m128i a,  int control);
+        ///   PSHUFHW xmm, xmm/m128, imm8
         /// </summary>
         public static Vector128<ushort> ShuffleHigh(Vector128<ushort> value, byte control) => ShuffleHigh(value, control);
-        
+
         /// <summary>
-        /// __m128i _mm_shufflelo_epi16 (__m128i a,  int control)
+        /// __m128i _mm_shufflelo_epi16 (__m128i a,  int control);
+        ///   PSHUFLW xmm, xmm/m128, imm8
         /// </summary>
         public static Vector128<short> ShuffleLow(Vector128<short> value, byte control) => ShuffleLow(value, control);
         /// <summary>
-        /// __m128i _mm_shufflelo_epi16 (__m128i a,  int control)
+        /// __m128i _mm_shufflelo_epi16 (__m128i a,  int control);
+        ///   PSHUFLW xmm, xmm/m128, imm8
         /// </summary>
         public static Vector128<ushort> ShuffleLow(Vector128<ushort> value, byte control) => ShuffleLow(value, control);
 
```

Comment:
Functions are not in alphabetical order - perhaps it would be easier to work with them if they will be sorted

Reply:
That can be dealt with by a separate PR.